crash-handler-module/main.js

```diff
@@ -44,9 +44,10 @@ function unregisterProcess(pid) {
 }
 
 async function terminateCrashHandler(pid) {
+    console.log('[crash-handler] terminateCrashHandler');
     const buffer = new Buffer.alloc(5);
     let offset = 0;
-    buffer.writeUInt8(2, offset++);
+    buffer.writeUInt8(3, offset++);
```

Comment:
adding a new action type here

Reply:
this function already existed, but instead sending action type 2 (which would correspond to memory dump), I send action type 3 which is terminate